### ISSUE TITLE
Introduce audioCues.volume setting

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#144771]
We should introduce the `audioCues.volume` setting. It should probably be a number from 1 to 10?
I am not sure how fine tuned you can set it in the web audio api

Default value should be in the middle as @gino-scarpino suggests

**Comment by isidorn** [comment]
@hediet let's not do this yet. Assigning to backlog. And once users request this more we can reconsider.
If you are passionate about this feel free to bring back to the Milestone :)

**Comment by hediet** [comment]
What should the default be? It set it to 50, but maybe 100 makes more sense.

**Comment by isidorn** [comment]
60?
I would hate that users find this setting and figure out it can not go any higher.
If we see that 60 the volume is still too law that means that the sound should be louder in the .opus file.

**Comment by Jonic** [comment]
I'd love to see a volume control. I'd also like to see the re-enabling of the "enable/disable all" setting alongside the granular controls for each.

However, for the default volume can I suggest `0`? I've just disabled all of the audio cues because I'm reviewing someone else's branch, and VS Code will not stop beeping at me. I have ADHD, and like many neurodiverse people I have auditory processing issues, which can make this kind of constant beeping very stressful.

To solve it I've disabled all of the sounds, and I likely won't be enabling them again. So the default setting here has resulted in me permanently disabling something which I may have found to be genuinely useful if it hadn't been so annoyingly stressful to hear all the time.

A better default experience would be to ask users if they want them enabled at all, and if they say yes to allow them to preview the volume before they commit. In future I think an option for a few different "sound palettes" would be a nice option too, because the sounds in use now feel too stabby to me!

I'm a little sad to see that this issue has been closed without a good reason why it's "not worth doing now", because I think the current situation is a definite accessibility issue. Given how good VS Code generally is with accessibility I hope that this issue will be reconsidered!

**Comment by isidorn** [comment]
@Jonic thanks a lot for your suggestions and feedback.
Let me clarify that we have closed this issue because we have done it, not because it is not worth doing. You can try it out in VS Code insiders from tomorrow and we would really appreciate your feedback.

I agree that the volume control should support `0`, but imho that should not be the default.
The current default imho is good = users do not hear anything unless they have a screen reader attached. If they have a screen reader attached only then they hear audio cues.
Having an option to set volume to 0 will give a master switch to disable sounds, and users can enable / disable each sound specifically. I believe this will prove to be a good user experience.

**Comment by isidorn** [comment]
@hediet yeah, then we need louder sounds as Gino mentioned in our chat.
Let's see with him if we get them this week. Would make this setting more useful.

**Comment by hediet** [comment]
Verification steps:
* Configure `audioCues.volume`. Default value is 50. Set to 10 or 100.
* Trigger an audio cue (e.g. through the audio cue help)
* Notice the volume is louder / quieter.

**Comment by jvesouza** [comment]
When the volume is at 100% and we press the up arrow, the orca announces the value of 101% followed by an error message. Something similar happens when the volume is at 0% and we press down arrow.

Would it be possible to use some key combination for bigger jumps in increment / decrement? Currently the jump is 1 when pressing down arrow or up arrow. Maybe ctrl+down arrow or ctrl+up arrow could produce an increment of 10.

**Comment by isidorn** [comment]
@jvesouza good point. But the settings UI does not allow us to have this fine grained change of values. fyi @rzhao271 
@hediet maybe that is the reason why we should keep it 1-10? Not sure.

**Comment by hediet** [comment]
Windows also uses 0-100 for the volume:

![image](https://user-images.githubusercontent.com/2931520/159498180-d0d4e658-7fb3-4a0d-af0b-0313f8c5db80.png)

VLC uses percent:
![image](https://user-images.githubusercontent.com/2931520/159498600-70d3df1b-e365-4482-bbca-7a6c51838992.png)


But I'm open for 0-10.

**Comment by jrieken** [comment]
Verified but I also wonder about the usefulness of `0-100`. I am unable to hear the difference between `37` and `38`...
